test/jasmine/tests/hover_label_test.js

```diff
@@ -2640,6 +2640,39 @@ describe('Hover on multicategory axes', function() {
         })
         .then(done, done.fail);
     });
+
+    it('should work with series', function(done) {
+        var fig = Lib.extendDeep({}, require('@mocks/zz-multicategory_series.json'));
+        fig.data = [fig.data[0]];
+        fig.layout.width = 500;
+        fig.layout.height = 500;
+
+        Plotly.newPlot(gd, fig)
+        .then(function() {
+            gd.on('plotly_hover', function(d) {
+                eventData = d.points[0];
+            });
+        })
+        .then(function() { _hover(200, 200); })
```

Comment:
Does this test pass on your machine?
On CircleCI we get different points when hovering over [200, 200] position.
Could you possibly simplify/adjust the test to pass?

Reply:
See https://app.circleci.com/pipelines/github/plotly/plotly.js/7672/workflows/447971ac-ba9c-46f3-af59-93ca0f1a3482/jobs/171830

Reply:
@archmoj I don't know why it's failing now